inspire/dojson/schemas/hep-0.0.1.json

```diff
@@ -0,0 +1,646 @@
+                                    "type": "string"
+                                },
+                                "value": {
+                                    "type": "string"
+                                }
+                            },
+                            "type": "object"
+                        }
+                    },
+                    "role": {
+                        "type": "string"
+                    },
+                    "recid": {
+                        "type": "integer"
+                    },
+                    "curated_curated": {
+                        "type": "bool"
+                    }
+                },
+                "type": "object"
+            },
+            "type": "array",
+            "uniqueItems": true
+        },
+        "experiment": {
+            "properties": {
+                "$ref": "#/definitions/relation"
+            },
+            "required": ["raw"],
+            "type": "object"
+        },
+        "book_series": {
+            "properties": {
+                "title": {
+                    "type": "string"
+                },
+                "volume": {
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        "classification_number": {
+            "items": {
+                "properties": {
+                    "value": {
+                        "type": "string"
+                    },
+                    "standard": {
+                        "type": "string"
+                    }
+                },
+                "type": "object"
+            },
+            "type": "array",
+            "uniqueItems": true
+        },
+        "collaboration": {
+            "items": {
+                "properties": {
+                    "": {
+                        "type": "string"
+                    }
+                },
+                "type": "object"
+            },
+            "type": "array",
+            "uniqueItems": true
+        },
+        "collections": {
+            "items": {
+                "type": "string"
+            },
+            "type": "array",
+            "uniqueItems": true
+        },
+        "corporate_author": {
+            "items": {
+                "type": "string"
+            },
+            "type": "array",
+            "uniqueItems": true
+        },
+        "copyright": {
+            "properties": {
+                "holder": {
+                    "type": "string"
+                },
+                "material": {
+                    "type": "string"
+                },
+                "statement": {
+                    "type": "string"
+                },
+                "url": {
+                    "format": "url",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        "dois": {
+            "items": {
+                "format": "doi",
```

Comment:
Here would have come handy the old integration of jsonalchemy. Should we create in `invenio-jsonschemas` registry a shared pool of regexp to be used as shared code? @tiborsimko, @egabancho, @jirikuncar?